core/primitives/src/block.rs

```diff
@@ -377,6 +38,12 @@ pub struct Block {
     pub vrf_proof: near_crypto::vrf::Proof,
 }
 
+/// Versioned Block data structure.
+#[derive(BorshSerialize, BorshDeserialize, Serialize, Debug, Clone, Eq, PartialEq)]
+pub enum Block {
+    BlockV1(Box<BlockV1>),
```

Comment:
I think it is good to have comments here going forward indicating the protocol version where the block structure is changed.